CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- **`Select`** New component.
```

Comment:
Não acha que podem confundir Select com o componente relativo a `<select>` do HTML?

Acho que os componentes Dropdown e Select vão ser bastante confundidos.

- Dropdown é o `<select>`
- Select é o (?) multiselect?

Reply:
@klzns esse novo Select substitui tanto o Dropdown e o Multiselect. Discutimos isso na última reunião do Styleguide, acho que tu tava lá até. A principio ambos podem deixar de ser usados.

Reply:
You can use this `Select` as a single option as well as a multi select option. Since the `MultiSelect` name is already taken, I thought `Select` would be fine... Any suggestion?

Reply:
I think we can start to think about deprecating the MultiSelect and maybe Dropdown since the Select component achieves the same goals. What do you guys think?

Reply:
IMO, if we were to deprecate `Dropdown`, we should at least implement the same mobile behavior it has (the native one).